src/github.com/getlantern/lantern/lantern.go

```diff
@@ -37,16 +37,15 @@ type SocketProtector interface {
 // because it keeps Lantern's own connections from being captured by the VPN and
 // resulting in an infinite loop.
 func ProtectConnections(dnsServer string, protector SocketProtector) {
-	protected.Configure(protector.Protect, dnsServer)
-	tlsdialer.OverrideResolve(protected.Resolve)
-	tlsdialer.OverrideDial(protected.Dial)
+	p := protected.New(protector.Protect, dnsServer)
```

Comment:
@myleshorton Because protected is only referenced from lantern now, and because lantern is only referenced from the mobile library, we no longer need to do platform detection to exclude protected from non-Android builds.